src/peripherals/usart.c

```diff
@@ -183,9 +184,9 @@ void usarts_enable(u32 ftdi_baud, u32 uarta_baud, u32 uartb_baud, bool do_precon
 
   if (do_preconfigure_hooks) {
 
-    printf("\n\nPiksi Starting...\n"
-       "Firmware Version: " GIT_VERSION "\n" \
-       "Built: " __DATE__ " " __TIME__ "\n");
+    log_info("Piksi Starting...\n");
```

Comment:
I feel like log_warn should be reserved for bad things only, it makes it easier for people to react to the warnings if they know there are none that should just be ignored. There are a couple options:

 - Ship with log level set to INFO
 - Don't worry about it because the same info can be found on the settings tab
 - Have the console detect the `MSG_STARTUP` message and print or show some other kind of indication of the Piksi booting up